src/components/Flows/__tests__/Flows.test.tsx

```
@@ -15,128 +15,1042 @@
+  //   const confirmButton = screen.getByRole('button', { name: /confirm/i });
+  //   await userEvent.click(confirmButton);
+
+  //   // Verify delete API was called
+  //   await waitFor(() => {
+  //     expect(httpDelete).toHaveBeenCalledWith(mockSession, 'prefect/v1/flows/delete-deployment-id');
+  //   });
+  // });
+
+  // ================================================================================
+  it('handles quick run deployment success', async () => {
+    const { httpPost } = require('@/helpers/http');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)